management/projects/update_issues_and_prs.sh

```diff
@@ -0,0 +1,377 @@
+#!/bin/bash
+
+# Command to update fetched issues and prs in a single repo in an org
+# update_repo.sh <file with issues> <file with prs> <org>
+
+echo "$0 $@"
+
+# Arguments are files to process
+ISSUE_FILE="${1}"
+PR_FILE="${2}"
+
+
+# Issues filter
+ISSUE_FILTER='.[].data.repository.issues.nodes[] | 
+""+.id+","+(.number|tostring)+","+.state+","+.createdAt+","+.updatedAt'
+# PR filter
+PR_FILTER='.[].data.repository.pullRequests.nodes[] | 
+""+.id+","+(.number|tostring)+","+.state+","+
+.createdAt+","+.updatedAt+","+
+(.isDraft|tostring)+","+
+(.merged|tostring)'
+
+
+# Project ID for Enarx
+ENARX_PROJECTID="PVT_kwDOAqzHpc4AIh12"
```

Comment:
Are any of the values in this file supposed to be private secrets? Where do all these IDs come from?

Reply:
Since the projects are public, anyone can look up these IDs for the projects themselves and the fields and options. One can use API, CLI or GraphQL Explorer to do that. So this information is not private.